docs/aws.md

```diff
@@ -0,0 +1,67 @@
+# AWS
+
+## Configuration
+
+### Example
+
+<!-- UPDATE THIS JSON EXAMPLE -->
+
+```json
+{
+  "settings": [
+    {
+      "provider": "aws",
+      "domain": "domain.com",
+      "host": "@",
+      "ip_version": "ipv4",
+      "ipv6_suffix": "",
+      "aws_access_key_id": "ffffffffffffffffffff",
+      "aws_secret_access_key": "ffffffffffffffffffffffffffffffffffffffff",
+      "hosted_zone_id": "A30888735ZF12K83Z6F00",
+      "region": "eu-central-1",
+      "ttl": 60
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"`
+- `"host"` is your host and can be a subdomain or `"@"` or the wildcard `"*"`
+- `"aws_access_key_id"`
+- `"aws_secret_access_key"`
+- `"hosted_zone_id"`
```

Comment:
Could this be replaced with `zone_id` perhaps? 🤔

Reply:
I tried using AWS specific names, I can replace this by `zone_id` and update the doc to reference the HostedZone ID

Reply:
Updated to `zone_id`. Please, take another look.